File: converter.py
import requests

class Converter():

    def __init__(self, old, new, amount):

        self.old = old
        self.new = new
        self.amount = amount

    def convert(self):

        url = 'https://api.exchangerate.host/convert?from={}&to={}&amount={}'.format(self.old, self.new, self.amount)
        response = requests.get(url)
        data = response.json()

        return data.get("result")
    
    # def symbol(self):


    
        

    

File: converter.js

const $toresults = $("#to");
const $fromresults = $("#from");
const $amount = $("#amount");


$("#currency_convert_form").on("submit", async function(e) {

   e.preventDefault();

  const response = await axios.get("https://api.exchangerate.host/symbols")

  let symbols = response.get("symbols");
  let symbol_keys = Object.keys(symbols);

  if (!symbol_keys.includes($fromresults)) {
    $(".old").text("Not valid currency")
    }

  if (!symbol_keys.includes($toresults)) {
        $(".new").text("Not valid currency")
        }

  if (typeof $amount == 'number') {
        $(".amount").text("Not valid number")
    }
} 



//#New function with the value - Append - referred to in the next flask route